how-to-participate-in-meetings.md

```diff
@@ -35,8 +35,9 @@ Committee consensus is required to advance a proposal to a subsequent stage and
     1. The presenter asks the committee for consensus on $xyz.
          1. (Note, the chair may encourage the presenter to make this request if it appears that the committee conversation has settled, or the presenter may decide to ask this directly.)
     1. Attendees are asked if they support the proposal advancing, giving space for people to explain why they want the proposal to advance. To proceed, at least one delegate must provide explicit support. The typical expectation is to hear from at least two people at this point, as a proposal should have broad, active support from the committee to advance.
-    1. The chair will then ask the committee, “Does anyone object to $xyz for consensus?”
-    1. If there are no objections, the chair states, “$xyz has achieved consensus”.
+    1. Attendees are asked if they have any concerns about the proposal. These concerns may or may not be objections to consensus. Explicit statements of "non-blocking" concerns (that is, concerns which are not full objections) are very much welcome, whether these are requests for changes at a future stage or simply doubt which does not rise to the level of wanting to stop the proposal from happening. If a concern expressed here is non-blocking, it is best to explicit identify it as such, to avoid misunderstandings.
+    1. If no blocking concern was raised, and the champion wishes to proceed in light of any concerns raised so far, the chair will then ask the committee, “Does anyone object to $xyz for consensus?”
+    1. If there are objections, the chair states, “$xyz has achieved consensus”.
```

Comment:
```suggestion
    1. If there are no objections, the chair states, “$xyz has achieved consensus”.
```

Reply:
lol very important catch, think of the consequences if we had landed this!